config/webpack.backend.dev.config.babel.js

```diff
@@ -1,37 +1,16 @@
-import path from 'path';
-import fs from 'fs';
-import config from './config';
+import webpack from 'webpack';
+import merge from 'webpack-merge';
+import webpackBase from './webpack.backend.base.config.babel';
 
-// http://jlongster.com/Backend-Apps-with-Webpack--Part-I
-const nodeModules = {};
-fs.readdirSync(config.nodeModules)
-  .filter((x) => {
-    return ['.bin'].indexOf(x) === -1;
-  })
-  .forEach((mod) => {
-    nodeModules[mod] = `commonjs ${mod}`;
-  });
-
-export default {
-  entry: [
-    `${config.backend}/server.js`,
+const webpackDev = merge(webpackBase, {
+  debug: true,
+  plugins: [
+    // https://webpack.github.io/docs/list-of-plugins.html#bannerplugin
+    new webpack.BannerPlugin('require("source-map-support").install();',
+      { raw: true, entryOnly: false, environment: 'node' }),
   ],
-  target: 'node',
-  output: {
-    path: path.join(config.root, '/dist/backend/'),
-    filename: 'server.js',
-  },
-  resolve: {
-    alias: config.alias,
-  },
-  externals: nodeModules,
-  module: {
-    preLoaders: [
-      { test: /\.js$/, exclude: /node_modules/, loader: 'eslint-loader' },
-    ],
-    loaders: [
-      { test: /\.js$/, exclude: /node_modules/, loaders: ['babel-loader'] },
-      { test: /\.json$/, exclude: /node_modules/, loader: 'json-loader' },
-    ],
-  },
-};
+  // https://webpack.github.io/docs/configuration.html#devtool
+  devtool: 'cheap-source-map',
```

Comment:
Yes. We will also have the `webpack.frontend.dev.config.babel.js` that "extends" `webpack.frontend.base.config.babel.js` and creates sourcemaps so that we can have proper error message stacks.